packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/components/Stages/Stage/Stage.js

```diff
@@ -171,14 +169,15 @@ export function Stage({
     ),
     color: hex,
   }));
-  // TODO: the .toUpperCase() conversion can be removed once the hex code is normalized in
-  // the admin API
-  const colorValue = colorOptions.find(({ value }) => value === colorField.value.toUpperCase());
 
   React.useEffect(() => {
     dragPreviewRef(getEmptyImage(), { captureDraggingState: false });
   }, [dragPreviewRef, index]);
 
+  const { themeColorName } = colorField.value ? getStageColorByHex(colorField.value) : {};
+
+  const colorValue = colorOptions.find(({ value }) => value === colorField.value.toUpperCase());
```

Comment:
Could you please add the comment back? It is still valid :)

Reply:
Ah yes, sorry i was supposed to do that 🤦🏼‍♀️